sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/FormTrainingClient.java

```diff
@@ -86,34 +83,31 @@ public FormRecognizerServiceVersion getServiceVersion() {
      * error message indicating absence of cancellation support.</p>
      *
      * <p><strong>Code sample</strong></p>
-     * {@codesnippet com.azure.ai.formrecognizer.FormTrainingClient.beginTraining#string-boolean-boolean-string-Duration}
-     *
-     * @param fileSourceUrl source URL parameter that is either an externally accessible Azure storage
-     * blob container Uri (preferably a Shared Access Signature Uri).
-     * @param useLabelFile Boolean to specify the use of labeled files for training the model.
-     * @param includeSubFolders to indicate if sub folders within the set of prefix folders will
-     * also need to be included when searching for content to be preprocessed.
-     * @param filePrefix A case-sensitive prefix string to filter documents in the source path
-     * for training. For example, when using a Azure storage blob Uri, use the prefix to restrict
-     * sub folders for training.
+     * {@codesnippet com.azure.ai.formrecognizer.training.FormTrainingClient.beginTraining#string-boolean-trainModelOptions-Duration}
+     *
+     * @param trainingFilesUrl an externally accessible Azure storage blob container Uri (preferably a
+     * Shared Access Signature Uri).
+     * @param useTrainingLabels Boolean to specify the use of labeled files for training the model.
+     * @param trainModelOptions Filter to apply to the documents in the source path for training.
      * @param pollInterval Duration between each poll for the operation status. If none is specified, a default of
      * 5 seconds is used.
      *
      * @return A {@link SyncPoller} that polls the extract receipt operation until it
      * has completed, has failed, or has been cancelled.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public SyncPoller<OperationResult, CustomFormModel> beginTraining(String fileSourceUrl, boolean useLabelFile,
-        boolean includeSubFolders, String filePrefix, Duration pollInterval) {
-        return client.beginTraining(fileSourceUrl, useLabelFile, includeSubFolders,
-            filePrefix, pollInterval).getSyncPoller();
+    public SyncPoller<OperationResult, CustomFormModel> beginTraining(String trainingFilesUrl,
```

Comment:
this is probably a question for the wider team, but should we include pollInterval in the TrainModelOptions? I feel like it's not necessarily related (since pollInterval is more about the LROPoller itself), but the user might be confused seeing this and wondering why pollInterval isn't a training model option? Not sure what your thoguhts are on this, I don't feel strongly either way, just wanted to bring it up.

I'm assuming useLabelFile is also not in TrainModelOptions since we want it more exposed to the user, right?

Reply:
>I'm assuming useLabelFile is also not in TrainModelOptions since we want it more exposed to the user, right?

Yup, API's are using `useTrainingLabels` to distinguish  between labeled and unlabeled return result so not containing it in an object and explicitly exposed.

> but should we include pollInterval in the TrainModelOptions? 
I am open to doing that. 
We could reconsider removing `pollInterval` as an option all together since we don't have a lot of supporting use cases to include it?
@kristapratico @annelo-msft

Reply:
Guidelines say we need to support it: https://azure.github.io/azure-sdk/general_design.html#long-running-operations

Where is `pollInterval` on the /Analyze methods? I would place it in the same spot across methods - which I think would be outside of `TrainModelOptions`, but correct me if I'm wrong.

Reply:
Yeah, it is not included in any Model.